jedi/evaluate/docstrings.py

```diff
@@ -115,13 +258,32 @@ def _strip_rst_role(type_str):
 
 
 def _evaluate_for_statement_string(evaluator, string, module):
-    code = dedent("""
+    """
+    >>> from jedi.evaluate.docstrings import *  # NOQA
+    >>> from jedi.evaluate.docstrings import _search_param_in_docstr
+    >>> from jedi.evaluate.docstrings import _evaluate_for_statement_string
+    >>> from jedi.evaluate.docstrings import _execute_array_values
+    >>> from jedi.evaluate.docstrings import _execute_types_in_stmt
```

Comment:
I don't really get why you use them in the first place. I would just remove it all. doctests serve as documentation IMO and this is more confusing than anything.